ext/hash/hash.stub.php

```diff
@@ -43,7 +43,11 @@ function mhash_keygen_s2k(int $hash, string $input_password, string $salt, int $
 
 function mhash_count(): int {}
 
-function mhash(int $hash, string $data, string $key = UNKNOWN): string|false {}
+/**
+ * @param int|string $hash
+ * @param string|bool $params
+ */
+function mhash($hash, ...$params): string|false {}
```

Comment:
In order to be able to re-add the `$data` parameter, could we add it to the first ZPP call?
```C
if (zend_parse_parameters(2, "zS", &z_algorithm, &data) == FAILURE) {
    RETURN_THROWS();
}
```